public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js

```diff
@@ -1296,7 +1296,7 @@ jQuery.Class('Vtiger_Widget_Js', {
 			if (height < 400) {
 				height = 400;
 			}
-			this.print(imgEl.get(0), title, width, height);
+			this.printImage(imgEl.get(0), title, width, height);
```

Comment:
print image is used in two places (DRY) , before there was this.print method but I have no idea where it sink